test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
Hrmmmm, this doesn't quite feel right. I feel like `price` alone is somewhat ambiguous. Also, it just feels odd looking at json with just the input/output quantity and not the totalPrice.

Just throwing this out there: What do you think about leaving `totalPrice` and setting the `price` as just a string field `rate`?  Another option is to frame `quality` as `output_rate`, as offers regardless of buy/sell direction, have constant inputs and outputs.